### ISSUE TITLE
Move Headers to Second Row

### ISSUE THREAD

**Issue by inquisitive-stha** [rap2hpoutre/fast-excel#196]
I am looking to add a file name on the First row and move the header row to another row like 2 or 3. Is it possible to move this header row?

**Comment by rap2hpoutre** [comment]
Hi @inquisitive-stha thank you for this suggestion. It would be an interesting feature indeed. I keep this ticket open, we could try to tackle this issue!

**Comment by BrutusBossNL** [comment]
What I did:

In "src\FastExcel" - added after existing "header code blocks":
```php
    /**
     * @var bool
     */
    private $with_header = true;

    /**
     * @var integer
     */
    private $start_row = 1;

// ...code

    /**
     * @return $this
     */
    public function withoutHeaders()
    {
        $this->with_header = false;

        return $this;
    }

    /**
     * @return $this
     */
    public function startRow(int $row)
    {
        $this->start_row = $row;

        return $this;
    }
```

In "src\Importable" - changed function "importSheet":
```php
    /**
     * @param SheetInterface $sheet
     * @param callable|null  $callback
     *
     * @return array
     */
    private function importSheet(SheetInterface $sheet, callable $callback = null)
    {
        $headers = [];
        $collection = [];
        $count_header = 0;

        if ($this->with_header) {
            foreach ($sheet->getRowIterator() as $k => $row) {
                if ($k >= $this->start_row) {
                    if ($k == $this->start_row) {
                        $headers = $this->toStrings($row);
                        $count_header = count($headers);
                        continue;
                    }
                    if ($count_header > $count_row = count($row)) {
                        $row = array_merge($row, array_fill(0, $count_header - $count_row, null));
                    } else if ($count_header < $count_row = count($row)) {
                        $row = array_slice($row, 0, $count_header);
                    }
                    if ($callback) {
                        if ($result = $callback(array_combine($headers, $row))) {
                            $collection[] = $result;
                        }
                    } else {
                        $collection[] = array_combine($headers, $row);
                    }
                }
            }
        } else {
            foreach ($sheet->getRowIterator() as $k => $row) {
                if ($k >= $this->start_row) {
                    if ($callback) {
                        if ($result = $callback($row)) {
                            $collection[] = $result;
                        }
                    } else {
                        $collection[] = $row;
                    }
                }
            }
        }

        return $collection;
    }
```

Unfortunately I'm not that far into GIT. I don't know how to create a merge request or something, I just change the code after an update. Totally wrong, I know. 

Could you test/review my code and eventually add it to your package? (so other could make use of start/skip rows)

**Comment by BrutusBossNL** [comment]
Alright, I dived into GIT. I created a PULL request for the code changes a made. Please feel free to check/test.

**Comment by rap2hpoutre** [comment]
Fixed via https://github.com/rap2hpoutre/fast-excel/pull/211 thanks to @BrutusBossNL available in v2.5.0
